.codecov.yml

```diff
@@ -3,3 +3,12 @@ comment: off
 parsers:
   javascript:
     enable_partials: yes
+coverage:
+  status:
+    patch:
+      default:
+        informational: false
+        threshold: 5.0
```

Comment:
I personally think 5% is a lot. There is the risk we miss out on crucial coverage; we ignore the check as it is _all green_.

But let us try it out and move from there. I may play around with this value in secrecy myself, too. Just to see when we reach the annoying threshold.

Reply:
We will still receive annotations on the files on github when lines are not covered. We can experiment and see what works